dev/io.openliberty.netty.internal.impl/src/io/openliberty/netty/internal/tcp/LibertyLoggingHandler.java

```diff
@@ -0,0 +1,161 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.netty.internal.tcp;
+
+import java.net.SocketAddress;
+
+import com.ibm.websphere.ras.Tr;
+import com.ibm.websphere.ras.TraceComponent;
+import com.ibm.websphere.ras.annotation.Trivial;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.channel.ChannelFuture;
+import io.netty.channel.ChannelFutureListener;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.channel.ChannelPromise;
+import io.netty.handler.logging.LoggingHandler;
+import io.netty.util.AttributeKey;
+
+@Trivial
+public class LibertyLoggingHandler extends LoggingHandler{
+
+	protected static final TraceComponent tc = Tr.register(LibertyLoggingHandler.class, new String[]{TCPMessageConstants.TCP_TRACE_NAME,TCPMessageConstants.NETTY_TRACE_NAME},
+			TCPMessageConstants.TCP_BUNDLE, LibertyLoggingHandler.class.getName());
+	
+	protected final static AttributeKey<Integer> BYTES_READ_KEY = AttributeKey.valueOf("BYTES_READ");
```

Comment:
Do you foresee this being used by any other class? It might be worth considering moving this definition to a place where it can be called into more generally within the netty internal package

Reply:
Don't think I see this being used anywhere else right now besides the TCP pipeline. The name of the class is misleading, I'll change to something more related to TCP to address this